Python/Product/Profiling/Profiling/StandaloneTargetView.cs

```diff
@@ -164,6 +164,9 @@ public StandaloneTargetView(IServiceProvider serviceProvider, StandaloneTarget t
             set {
                 if (_interpreterPath != value) {
                     _interpreterPath = value;
+                    if (Interpreter.Name == Strings.LaunchProfiling_OtherInterpreter && Interpreter.Path == null) {
```

Comment:
I don't think interpreter names are intended to be unique, so they shouldn't be used for checks like these. It's what `Id` is supposed to be for, but you don't always get that here...

So, I think it would be better to somehow mark the `PythonInterpreterView` for the "Other" case. The most straightforward way to model this would be to derive `CustomPythonInterpreterView` - then you can use that with `is` to check. You could then also replace this:
```
_customInterpreter = new PythonInterpreterView(Strings.LaunchProfiling_OtherInterpreter, "", null);
```
with just:
```
_customInterpreter = new CustomPythonInterpreterView();
```
and move the argument values into the constructor for `CustomPythonInterpreterView`.

Reply:
In fact, better yet, you could move any logic that's conditional on whether it's a regular or a custom interpreter into the interpreter view class, making them virtual methods. For example, instead of the conditional in `ProfileStandaloneTarget`, you can have `virtual GetInterpreterConfiguration()` in `PythonInterpreterView`, and override it in `CustomPythonInterpreterView`.

Reply:
Hi @int19h , I understand you want a separate class for the custom python interpreter, but I might still need to call the `CustomPythonInterpreterView` setter inside of the `_interpreterpath` setter to set the properties. Or do you have better suggestions? Thanks.

Reply:
Since the property values don't have to be computed from context, you can just move them to the base constructor call:
```cs
class CustomPythonInterpreterView : PythonInterpreterView {
    CustomPythonInterpreterView():
        base(Strings.LaunchProfiling_OtherInterpreter, "", null) {
    }
}
```

Reply:
@int19h Just updated the PR with `CustomPythonInterpreterView` 😄.